test/studies/ml/lib/MNISTTester.chpl

```diff
@@ -112,10 +118,27 @@ proc train(ref network,
             numCorrect += acc_;
         }
 
-        writeln("End of epoch ", epoch + 1,
-                " Loss ", decFormat(loss / testingData.size,2),
-                " Accuracy ", numCorrect, 
-                " / ", testingData.size);
+        if !justValidate then writeln("End of epoch ", epoch + 1,
+                                        " Loss ", decFormat(loss / testingData.size,2),
+                                        " Accuracy ", numCorrect, 
+                                        " / ", testingData.size);
+
+        if justValidate && epoch == numEpochs - 1 {
+            const accuracy = numCorrect:real / (testingData.size:real);
+            const epochLoss = loss / (testingData.size:real);
+            const accWindow = (10.0 / testingData.size:real);
```

Comment:
Why is the accuracy window scaled by the size of the training data set? Are you expecting variance in accuracy to decrease as the number of samples increases?

Reply:
Yes, that was my thinking.

Reply:
For 2000 images for example, this value will be 0.005 right? Which seems too small given that we were seeing a 1/100 change in accuracy on the test that failed. Or am I misunderstanding something?

Reply:
I believe the change of 1/100 is just for the tests that have a test dataset size of 100. If the testing size is 2000, then changing 1 would have an accuracy difference of 1/2000, and since `abs(expectedAcc - actualAcc) = 1/2000 < 10/2000`, the condition would hold.

Reply:
Ohhhh okay, I thought the accuracy was always a percent value for some reason, my bad. Disregard my comment, I think this is fine then.